### ISSUE TITLE
[Feature Request] Make deafness from pressure not so silent

### ISSUE THREAD

**Issue by MaxIsJoe** [unitystation/unitystation#9478]
### There is no other issue with the same feature request.

- [X] I have searched the existing issues

### Why is this feature needed/good for the game?

Immersion.

### Describe your feature in detail.

Deafness doesn't mean that all noise is 100% gone. When people sit in total silence, they still hear things like their bloodflow and heart-beat more clearly and can even experience tinnitus even if they're completely deaf.

When losing sound due to high/low pressure from space or other external sources, there should be a clear indicator that tells the player that they're losing hearing of their surroundings via sound as well. This can be done via sounds like heart beats and ringing that play corresponding to what the player is experiencing right now.

### User stories

You stand next to an explosion as it gets set off. The next thing you hear is ringing.. Followed by your own heartbeat as you survive the blast and enter a critical state.

------

You walk into the vacuum of space to fix wiring from the outside. As you sit there alone outside, your senses start to heighten and you start having the ability to clearly hear your own heartbeat and some white noise that may be your blood flow or your own breath inside your helmet.

**Comment by corp-0** [comment]
This could be solved by instead of modifying intensity of the sound, you make the sound more muffled the less pressure/ear damage. You will only hear very low pitch sounds then.

Also, a correction for your user story.
User stories always have the following pattern: as a USER_ROLE I want to WANT so that REASON.

As a player I want low pressure/ear damage to sound muffled instead of silent so that the complete silence isn't that immersion breaking as also represent better how low frequency vibrations can still be felt.

**Comment by eurtod** [comment]
I agree with this, along with it I think when wearing a hardsuit with oxygen you should be able to hear some sounds. Maybe just sounds with a lesser range so only things nearby?
